dump.go

```diff
@@ -425,6 +428,19 @@ func Sdump(value ...interface{}) string {
 	return (&Config).Sdump(value...)
 }
 
+// Dump a value to stdout in color
+func DumpColor(value ...interface{}) {
+	result := (&Config).Sdump(value...)
+	if err := quick.Highlight(os.Stdout, result, "go", "terminal256", "monokai"); err != nil {
+		os.Stdout.Write([]byte("[syntax highlight error: " + err.Error() + "]"))
+	}
+}
+
+// Dump a value to stdout in colour
+func DumpColour(value ...interface{}) {
```

Comment:
This two function names are kind of confusing.

Reply:
There are two spellings of colour/color, depending on where in the world you are. (I'm in Australia, we have a "u").